src/beacon-api/app/services/device_service.py

```
@@ -981,14 +981,14 @@ async def sync_devices(db: Session, token: str) -> Dict[str, Any]:
             all_devices.extend(data.get("devices", []))
             
             meta = data.get("meta", {})
-            total_items = meta.get("total", 0)
+            total_pages = meta.get("totalPages", 1)
             actual_limit = meta.get("limit", 100)
             
             # Fetch remaining pages based on actual limit and total items
             if total_items > actual_limit:
```

Ruff (0.15.9)

[error] Undefined name `total_items`

(F821)